bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -556,9 +565,7 @@ Error LinuxKernelRewriter::readORCTables() {
     BC.MIB->addAnnotation(*Inst, "ORC", Entry.ORC);
   }
 
-  // Older kernels could contain unsorted tables in the file as the tables  were
-  // sorted during boot time.
-  llvm::sort(ORCEntries);
+  outs() << "BOLT-INFO: parsed " << NumORCEntries << " ORC entries\n";
```

Comment:
Sorry about late comment but we should be using `BC.outs()` after https://github.com/llvm/llvm-project/pull/81524